userInterface.js

```diff
@@ -1,27 +1,43 @@
 var blessed = require('blessed');
+var colorPal = require('./themes.js');
+var supportsColor = require('supports-color');
 
+// Colors
+var colorSupportLevel = supportsColor ? supportsColor.level : 0;
+var defaultThemesByColorLevel = {
+    3: colorPal.default,
+    2: colorPal.default, // Not tested yet
+    1: colorPal.tty16Default,
+    0: colorPal.tty1Default // Can be "tested" by passing --no-color argument
+};
+
+var theme = defaultThemesByColorLevel[colorSupportLevel];
+var colors = theme.palette;
```

Comment:
- Yes, this could be in its own little module
- I also prefer the has256 and has16m. The thing is that they are "independant" flags which doesn't fit very well the idea of having a single "supported color" field that we can compare, filter, sort... What if I turn it into constants instead? For instance: `COLORLEVEL_MONO, COLORLEVEL_16, COLORLEVEL_256, COLORLEVEL_16M` or any similar names. The level field in the themes could be turned into something like `color: "mono" | "16" | "256" | "16m"` so the theme designers would find it very easy to understand. Tell me your idea about this
- For the commas, I can revert the change. I was not expecting ending up in a pull request :)